segmentation/include/pcl/segmentation/impl/extract_labeled_clusters.hpp

```diff
@@ -39,45 +39,99 @@
 
 #include <pcl/segmentation/extract_labeled_clusters.h>
 
+namespace impl
+{
+  // convert labeled_cluster_map_t
+  void
+  labeled_cluster_map_to_labeled_cluster_vector( const pcl::PCLHeader& header, pcl::labeled_cluster_map_t labeled_map, std::vector<std::vector<pcl::PointIndices>>& vec )
```

Comment:
Are we to assume the user may be providing a preallocated `std::vector<std::vector<pcl::PointIndices>>` which actually might contain pre-existing values?  Or just an empty vector that we can modify as we like?  In the case of the former, I was trying to avoid having to merge data, so I went with an output parameter instead.  If the latter, then I agree with you

Reply:
Good point. The documentation doesn't mention it, but assuming [Hyrum's law](https://www.hyrumslaw.com/), someone uses it to pass a non-empty vector.